UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
Yes, since login isn't currently implemented (could have done so without auth, but didn't get to it)